### ISSUE TITLE
New projects do not have script resource

### ISSUE THREAD

**Issue by codebling** [serge-rider/dbeaver#149]
A script resource is not created for new projects in the project view. As a result, it is impossible to open an SQL Editor on any connection in any project aside from the default project. 

A script resource/folder should be created for the project at project creation time. This will facilitate transfer of scripts between projects.

**Comment by serge-rider** [comment]
Scripts folder will be created on demand - once you will create new SQL script for any connection (CTRL+L, F3, CTRL+ALT+L).
Auto creation was disabled because of Eclipse plugin mode (usually you don't need Scripts folder in every Eclipse project you have).
Maybe it should be enabled by default in standalone mode..

**Comment by codebling** [comment]
I did notice that it worked on demand, but if you try to open an SQL Editor from a non-active project, it will fail. (At least as I recall. I am having issues launching DBeaver right now, I have probably borked a setting with all of my tinkering and testing)

If, from non-active projects with no Script folder, it is not possible to open a new SQL Editor tab (because the Script folder is not automatically created), then this should probably be a bug and not an enhancement.

**Comment by kjvalencik** [comment]
I get `2016-04-20 13:15:43.378 - org.eclipse.core.internal.resources.ResourceException: Resource '/General/Scripts' does not exist.` when trying to open a new sql script.

I needed to `mkdir ~/.dbeaver/General/Scripts` to get past this.

**Comment by serge-rider** [comment]
Please post error stacktrace (from Error Log view)

**Comment by kjvalencik** [comment]
```
2016-04-20 13:15:37.308 - DBeaver 3.6.5 is starting
```

```
2016-04-20 13:15:43.378 - org.eclipse.core.internal.resources.ResourceException: Resource '/General/Scripts' does not exist.
org.eclipse.core.internal.resources.ResourceException: Resource '/General/Scripts' does not exist.
	at org.eclipse.core.internal.resources.Resource.checkExists(Resource.java:334)
	at org.eclipse.core.internal.resources.Resource.checkAccessible(Resource.java:208)
	at org.eclipse.core.internal.resources.Container.members(Container.java:238)
	at org.eclipse.core.internal.resources.Container.members(Container.java:225)
	at org.jkiss.dbeaver.ui.resources.ResourceUtils.findScriptTree(ResourceUtils.java:168)
	at org.jkiss.dbeaver.ui.editors.sql.handlers.OpenSQLEditorHandler.execute(OpenSQLEditorHandler.java:64)
	at org.eclipse.ui.internal.handlers.HandlerProxy.execute(HandlerProxy.java:295)
	at org.eclipse.ui.internal.handlers.E4HandlerProxy.execute(E4HandlerProxy.java:90)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.eclipse.e4.core.internal.di.MethodRequestor.execute(MethodRequestor.java:56)
	at org.eclipse.e4.core.internal.di.InjectorImpl.invokeUsingClass(InjectorImpl.java:252)
	at org.eclipse.e4.core.internal.di.InjectorImpl.invoke(InjectorImpl.java:234)
	at org.eclipse.e4.core.contexts.ContextInjectionFactory.invoke(ContextInjectionFactory.java:132)
	at org.eclipse.e4.core.commands.internal.HandlerServiceHandler.execute(HandlerServiceHandler.java:152)
	at org.eclipse.core.commands.Command.executeWithChecks(Command.java:493)
	at org.eclipse.core.commands.ParameterizedCommand.executeWithChecks(ParameterizedCommand.java:486)
	at org.eclipse.e4.core.commands.internal.HandlerServiceImpl.executeHandler(HandlerServiceImpl.java:210)
	at org.eclipse.e4.ui.workbench.renderers.swt.HandledContributionItem.executeItem(HandledContributionItem.java:799)
	at org.eclipse.e4.ui.workbench.renderers.swt.HandledContributionItem.handleWidgetSelection(HandledContributionItem.java:675)
	at org.eclipse.e4.ui.workbench.renderers.swt.HandledContributionItem.access$7(HandledContributionItem.java:659)
	at org.eclipse.e4.ui.workbench.renderers.swt.HandledContributionItem$4.handleEvent(HandledContributionItem.java:592)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:84)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4481)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1329)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:3819)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3430)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$4.run(PartRenderingEngine.java:1127)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:337)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1018)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:156)
	at org.eclipse.ui.internal.Workbench$5.run(Workbench.java:694)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:337)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:606)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:150)
	at org.jkiss.dbeaver.core.application.DBeaverApplication.start(DBeaverApplication.java:110)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:380)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:235)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:669)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:608)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1515)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1488)

```

**Comment by serge-rider** [comment]
This was fixed in #802
